.env.example

```diff
@@ -1 +0,0 @@
-MONGOHQ_URL=your-database-url
```

Comment:
Can you add a newline to the end of this file?

Reply:
yes

Reply:
This error is actually from @larrystone